src/routes/+layout.svelte

```diff
@@ -1,12 +1,28 @@
-<script>
+<script lang="ts">
 	import '$lib/styles.scss';
 	import { page } from '$app/stores';
+	import Navbar from '$lib/Navbar.svelte';
+	import logoUrl from './logo.webp';
+
+	const pages = [
+		{ name: 'Calendar', url: '/calendar' },
+		{ name: 'About', url: '/about' },
```

Comment:
I noticed this page is called "about us" on the page itself but it only says "about" here. Should we make that consistent?

Reply:
Yeah we should, although part of me feels like it's weird to have "About Us" and "Contact Us" right on top of each other, like the lengths of those are so close that they look awkward next to each other, but maybe that's just me?
Still, it makes more sense than a different alternative, I think, and we're adding at least one more page anyway and possibly removing the Contact Us page, so it should be fine.
Changed page name to "About Us" in [68bc3d5](https://github.com/kendoclub-at-umich/website/pull/40/commits/68bc3d59f58cf413d92294f849b8e017564f3fbf)

Reply:
In general I would say content should take priority over styling but it's good that your thinking about these things (as long as you aren't overthinking 😉)